src/effects/VST/VSTEffect.h

```diff
@@ -563,4 +563,12 @@ class VSTEffectsModule final : public PluginProvider
       LoadPlugin(const PluginPath & path) override;
 };
 
+
+class VSTEffectValidator final : public DefaultEffectUIValidator
+{
+public:
+    VSTEffectValidator(EffectUIClientInterface& effect, EffectSettingsAccess& access);
```

Comment:
Might simplify it with an inherited constructor:
`using DefaultEffectUIValidator::DefaultEffectUIValidator `

Reply:
ok, done